bootloader/src/pyi_win32_utils.c

```diff
@@ -63,20 +63,27 @@ char * GetWinErrorString(DWORD error_code) {
     if(error_code == 0) {
         error_code = GetLastError();
     }
-    FormatMessageA(FORMAT_MESSAGE_FROM_SYSTEM, // dwFlags
-                   NULL,                       // lpSource
-                   error_code,                 // dwMessageID
-                   0,                          // dwLanguageID
-                   errorString,                // lpBuffer
-                   ERROR_STRING_MAX,           // nSize
-                   NULL                        // Arguments
-                   );
-
-    if (NULL == errorString) {
+    wchar_t local_buffer[ERROR_STRING_MAX / 3];
+    DWORD result;
+    char * result2;
+    result = FormatMessageW(
+        FORMAT_MESSAGE_FROM_SYSTEM, // dwFlags
+        NULL,                       // lpSource
+        error_code,                 // dwMessageID
+        MAKELANGID(LANG_NEUTRAL, SUBLANG_DEFAULT), // dwLanguageID
+        local_buffer,               // lpBuffer
+        ERROR_STRING_MAX / 3,       // nSize
+        NULL                        // Arguments
+        );
+
+    if (!result) {
         return "FormatMessage failed.";
     }
+    result2 = pyi_win32_utils_to_utf8(errorString, local_buffer, ERROR_STRING_MAX);
+    if (!result2) {
+        return "pyi_win32_utils_to_utf8 failed.";
```

Comment:
* Hmmm. But the code below from here:

      if (0 == ret) {
          FATAL_WINERROR("WideCharToMultiByte", "Failed to encode filename as ANSI.\n");
          return NULL;
      }

  You mean just use `FATAL_WINERROR` ?

* When `pyi_win32_utils_to_utf8` fails, `FATAL_WINERROR` called before returned from the function and no string returns. I think `GetWinErrorString` should not return empty string, how about it do you think?

* I think this is usable information(even not 'useful' though), because the true end users don't know the .exe file is built with PyInstaller. S/he will report to developers this message, and the developers find where the accident occurs. Of course that's very rare case and maybe no way to fix, how do you think besides them?